CommonProtos/README.md

```diff
@@ -0,0 +1,41 @@
+# Google Cloud PHP Common Protos
+
+[![Build Status](https://api.travis-ci.org/googleapis/google-cloud-php-common-protos.svg?branch=master)](https://travis-ci.org/googleapis/google-cloud-php-common-protos)
+
+This repository is a home for the [protocol buffer][protobuf] types which are
+shared by multiple Google Cloud APIs, generated for PHP.
+The protobuf definitions for these generated PHP classes are provided in the
+[Googleapis][googleapis] repository.
+
```

Comment:
Sorry, I just thought of one more thing. Could you add a note to the README directing users to Google Cloud PHP for bugs and support?

```
**NOTE:** This repository is part of [Google Cloud PHP](https://github.com/googleapis/google-cloud-php). Any
support requests, bug reports, or development contributions should be directed to
that project.
```

See [google/cloud-core/README.md](https://github.com/googleapis/google-cloud-php-core/blob/master/README.md) for example.

Reply:
Good catch, done!